pkg/scripts/poststart/check_fuse_app.go

```diff
@@ -20,61 +20,76 @@ import (
 	"fmt"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 	utilpointer "k8s.io/utils/pointer"
-	"strings"
 )
 
 const (
-	appScriptName = "check-dataset-ready.sh"
-	appScriptPath = "/" + appScriptName
-	appVolName    = "check-dataset-ready"
+	appScriptName    = "check-fluid-mount-ready.sh"
+	appScriptPath    = "/" + appScriptName
+	appVolName       = "check-fluid-mount-ready"
+	appConfigMapName = appVolName
 )
 
+var contentCheckMountReadyScript = `#!/bin/bash
+
+set -ex
+
```

Comment:
Please check the format of arguments. If the input is illegal, the script should return at once.